packages/react-core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -19,13 +16,15 @@ export enum TooltipPosition {
   right = 'right'
 }
 
-export interface TooltipProps {
+export interface TooltipProps extends Omit<React.HTMLProps<HTMLDivElement>, 'content'> {
   /** The element to append the tooltip to, defaults to body */
-  appendTo?: Element | ((ref: Element) => Element);
-  /** Aria-labelledby or aria-describedby for tooltip */
-  aria?: 'describedby' | 'labelledby';
-  /** If enableFlip is true, the tooltip responds to this boundary */
-  boundary?: 'scrollParent' | 'window' | 'viewport' | HTMLElement;
```

Comment:
Isn't removing this a breaking change?  Can we just leave it for now and deprecate it?   WE can open up a followup breaking change issue for our next major release.

Reply:
it technically is, but it shouldn't be needed any longer since the popper element can be appended to any container, I tried to re-create the functionality but with the new popper version it didn't work really well. Even the recent tippy versions have removed it. 
I figured, this close to the last breaking change release it might be okay to remove this mostly useless option! 
Of course, I could add it back but not let it do anything if that's what you mean?

Reply:
I think that would be best incase anyone is using it.

Reply:
If we do we should warn that the prop is ignored.

Reply:
@tlabaj @redallen I added it back as a deprecated prop for both Tooltip and Popover and have also added a warning